srcjs/init_shiny.js

```diff
@@ -421,10 +421,20 @@ function initShiny() {
   });
 
   // This is needed for Bootstrap 2 compatibility and for non-Bootstrap
-  // related shown/hidden events (like conditionalPanel)
+  //   related shown/hidden events (like conditionalPanel)
+  // Execute on all body inner elements
   $(document.body).on('shown.sendImageSize', '*', sendImageSize);
-  $(document.body).on('shown.sendOutputHiddenState hidden.sendOutputHiddenState', '*',
-               sendOutputHiddenState);
+  $(document.body).on(
```

Comment:
If you replace `$(document.body)` with `$(document)`, then you can remove the duplicated calls below (the ones without the `'*'`. I think that should work fine without any drawbacks, but it wouldn't hurt to check.

Reply:
Checked.  It wouldn't hurt as we have the fns debounced, but it's called twice.

```js
$(document).on("schloerke", "*", function(){console.log("schloerke")})
// jQuery.fn.init [document, context: document]
$(document).on("schloerke.wch", "*", function(){console.log("schloerke.wch")})
// jQuery.fn.init [document, context: document]
$(document.body).trigger("schloerke")
// VM1091:1 schloerke
// VM1125:1 schloerke.wch
// VM1091:1 schloerke
// VM1125:1 schloerke.wch
// jQuery.fn.init [body.api.jquery.single.single-post.postid-461.single-format-standard.single-author.singular, context: body.api.jquery.single.single-post.postid-461.single-format-standard.single-author.singular]
```

Even then, I'd like to clean up the trigger request to `$(document).trigger("shown")`... which makes me think it should be a standardized Shiny.js method.